osgp/platform/osgp-adapter-ws-shared/src/main/java/org/opensmartgridplatform/adapter/ws/infra/jms/LoggingMessageSender.java

```diff
@@ -50,28 +56,51 @@ private void sendMessage(final LoggingRequestMessage loggingMessage) {
 
     this.loggingJmsTemplate.send(
         new MessageCreator() {
+
           @Override
           public Message createMessage(final Session session) throws JMSException {
-            final ObjectMessage objectMessage = session.createObjectMessage();
-            objectMessage.setJMSCorrelationID(loggingMessage.getCorrelationUid());
-            objectMessage.setLongProperty(
-                Constants.TIME_STAMP, loggingMessage.getTimeStamp().getTime());
-            objectMessage.setStringProperty(Constants.CLASS_NAME, loggingMessage.getClassName());
-            objectMessage.setStringProperty(Constants.METHOD_NAME, loggingMessage.getMethodName());
-            objectMessage.setStringProperty(
-                Constants.ORGANISATION_IDENTIFICATION,
-                loggingMessage.getOrganisationIdentification());
-            objectMessage.setStringProperty(Constants.USER_NAME, loggingMessage.getUserName());
-            objectMessage.setStringProperty(
-                Constants.APPLICATION_NAME, loggingMessage.getApplicationName());
-            objectMessage.setStringProperty(
-                Constants.DEVICE_IDENTIFICATION, loggingMessage.getDeviceIdentification());
-            objectMessage.setStringProperty(
-                Constants.RESPONSE_RESULT, loggingMessage.getResponseResult());
-            objectMessage.setIntProperty(
-                Constants.RESPONSE_DATA_SIZE, loggingMessage.getResponseDataSize());
-            return objectMessage;
+            if (LoggingMessageSender.this.isCreateJsonMessage()) {
+              return LoggingMessageSender.this.getJsonMessage(loggingMessage, session);
+            }
+            return LoggingMessageSender.this.getObjectMessage(session, loggingMessage);
```

Comment:
LoggingMessageSender. can be removed 3 times

Reply:
really weird, this comes somehow from the formatting rules on save.